docs/config.adoc

```diff
@@ -30,18 +30,19 @@ The following environment variables can be used to affect some standard Django s
 - `EMAIL_URL`: Configures email for Django using https://django-environ.readthedocs.io/en/latest/#email-settings[Django-environ]. Default is "consolemail://".
 - `VAR_ROOT`: Provides a base path for the https://docs.djangoproject.com/en/2.2/ref/settings/#media-root[`MEDIA_ROOT`] and https://docs.djangoproject.com/en/2.2/ref/settings/#static-root[`STATIC_ROOT`] Django settings. `MEDIA_ROOT` will be set to `${VAR_ROOT}/media` and `STATIC_ROOT` to `${VAR_ROOT}/static`. If run with Docker (determined by checking that the project is in an `/app` directory), `VAR_ROOT` defaults to `/var`. Otherwise it defaults to a path called `var` under the project directory.
 
-== Authentication server(s)
+== Authentication and authorization
 
 This project uses https://github.com/City-of-Helsinki/django-helusers[Django-helusers] for doing authentication. The following environment variables correspond directly to Django-helusers settings:
 
 - `TOKEN_AUTH_ACCEPTED_AUDIENCE`: Corresponds to `OIDC_API_TOKEN_AUTH.AUDIENCE`. Default is empty.
 - `TOKEN_AUTH_ACCEPTED_SCOPE_PREFIX`: Corresponds to `OIDC_API_TOKEN_AUTH. API_SCOPE_PREFIX`. Default is empty.
 - `TOKEN_AUTH_REQUIRE_SCOPE`: Corresponds to `OIDC_API_TOKEN_AUTH. REQUIRE_API_SCOPE_FOR_AUTHENTICATION`. Default is `False`.
 
-The following environment variables configure authentication in other ways:
+The following environment variables configure authentication or authorization in other ways:
 
 - `TOKEN_AUTH_AUTHSERVER_URL`: Sets the "main" authentication server's URL. The URL needs to be exactly what the authentication server reports as its `issuer` value. Default is empty.
 - `ADDITIONAL_AUTHSERVER_URLS`: Sets additional authentication server URLs as a list of strings. JWTs signed by these servers are accepted for authentication. The URLs need to be exactly what the authentication servers report as their `issuer` value. Default is empty list.
+- `VERIFIED_PERSONAL_INFORMATION_ACCESS_AMR_LIST`: Can be used to limit staff users access to verified personal information fields for only those that have authenticated using certain authentication method. If empty, access is not limited. Default is empty list.
```

Comment:
Perhaps this could mention that the authentication method corresponds to the JWT's amr claim.